src/components/ui/phone-input.tsx

```diff
@@ -120,7 +120,11 @@ const CountrySelect = ({
           />
         </Button>
       </PopoverTrigger>
-      <PopoverContent className="w-[300px] p-0">
+      <PopoverContent
+        className="p-2 sm:p-0 w-[var(--radix-popover-trigger-width)] min-w-[250px]"
```

Comment:
let's NOT use arbitrary pixel values when it's possible without it!
```suggestion
        className="p-2 sm:p-0 w-[var(--radix-popover-trigger-width)] min-w-64"
```

Reply:
do check and verify things before requesting next review